confdb/aclocal_cc.m4

```diff
@@ -827,6 +829,8 @@ dnl     sixteen
 dnl
 dnl In addition, a "Could not determine alignment" and a "error!"
 dnl return is possible.  
+dnl 
+dnl Note: this macro assumes AC_CHECK_SIZEOF([long double]) has been run
```

Comment:
We should rerun `AC_CHECK_SIZEOF([long double])` instead of assuming.

Reply:
The intention is more about setting clear stages in `configure`. Otherwise, support of configure options and support cross-files become messy. Will it be more acceptable if we set stage-variable and assert the stage variable here rather than to assume in a comment?

Reply:
The assumption in the comment is not a good choice.  If you don't want to rerun sizeof, then we should at least assert.

Reply:
Agreed.